source/renderer/index.ejs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="utf-8">
+    <!--
+        unsafe-eval is require to run WASM code
```

Comment:
Why do we have WebAssembly code at all?

I asked this question above, what is this file?

```
dist/renderer/7ae194443b56ad7c5250.module.wasm
```